tests/manager/manager-ctor_error-003.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): read preference options of the wrong type will throw an InvalidArgumentException exception
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$invalidOptions = array(
+    [],
+    123,
+    2.5,
+    0x00,
+    null,
+    true
+);
```

Comment:
I might suggest just adapting the `manager-ctor-read_preference-003.phpt` test case here. I don't think it's necessary to test all possible types here, as one invalid type for each option should be sufficient to exercise the validation logic.